client/components/TestQueueRow/index.jsx

```diff
@@ -120,6 +127,24 @@ const TestQueueRow = ({
                 Published {gitUpdatedDateToString(testPlanVersion.updatedAt)}
             </p>
         );
+
+        const latestTestPlanVersion = latestTestPlanVersions.filter(
+            version => version.latestTestPlanVersion.id === testPlanVersion.id
+        );
+        const updateTestPlanVersionButton =
+            isAdmin && latestTestPlanVersion.length === 0 ? (
+                <Button
+                    className="updater-button"
+                    onClick={() => setShowTestPlanUpdaterModal(true)}
+                    size="sm"
+                    variant="outline-secondary"
```

Comment:
My initial feeling about this visually was that I didn't expect it could be interacted with because of the variant. Would be curious to know if anyone else had the same impression. But if I'm alone in my thoughts here, then no problem!